rust/kernel/sync/guard.rs

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0
+
```

Comment:
```suggestion

//! Lock guard.

```

i.e. please document the modules themselves, even if not public -- the idea is to use it as a "file information header".

Reply:
Done.